rot13/main.py

```diff
@@ -9,39 +9,6 @@
 	autoescape=True)
 
 
-def encode_algo(character):
-	# save case of character, convert to lower if upper
-	lower_case = True
-	if character.isupper():
-		lower_case = False
-		character = character.lower()
-
-	alphabet = 'abcdefghijklmnopqrstuvwxyz'
-	idx = alphabet.index(character)
-	if idx <= 12:
```

Comment:
This is the same as 
```python
code_idx = (alphabet.index(character) + 13) % 26
```

Modulus is a nice way to think about rotations.

You could also write it:
```python
idx = alphabet.index(character)
code_idx =  idx + 13 % alphabet.length
```